src/Layout/Header/Tools/about-modal.tsx

```diff
@@ -80,6 +82,14 @@ const AboutModalItems: React.FC<AboutModalItemsProps> = (
               </TextListItem>
             </>
           )}
+          {browserName && (
```

Comment:
Personally I would change browser info order in the following way: name, version, OS.
WDYT?

Reply:
I've just updated the order in the latest commit